lib/email/styles.rb

```diff
@@ -228,7 +228,7 @@ def replace_relative_urls
 
       @fragment.css('[href]').each do |element|
         href = element['href']
-        if href =~ /^\/\/#{host}/
+        if href.start_with?("\/\/#{host}")
```

Comment:
`start_with?` is faster https://github.com/JuanitoFatas/fast-ruby#stringmatch-vs-stringstart_withstringend_with-code-start-code-end

I was curious as well so I wrote this script

```
require 'memory_profiler'

test_string = 'abcde'
abc = "abc"
MemoryProfiler.report do
  test_string.start_with?(abc)
  # test_string =~ /\/\/#{abc}/
end.pretty_print
```

`start_with?` doesn't allocate any objects while using Regexp allocates 4 strings for this case.